src/description.jl

```diff
@@ -32,6 +36,8 @@ function description(dt::AbstractDateTime, p::Period, suffix::String)
         ts = string(max_val)
     end
 
+    isa(dt, Time) && return "$(prefix(p))$suffix $ts"
```

Comment:
I'd similarly make this `string(prefix(p), suffix, ' ', ts)` for consistency, though this case is easier to read than the previous.